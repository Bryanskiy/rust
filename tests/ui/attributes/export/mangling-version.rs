//@ build-fail
//@ compile-flags: -Zunstable-options -Csymbol-mangling-version=legacy
#![crate_type = "dylib"]
#![feature(export)]

#[export]
pub extern "C" fn foo() -> i32 { 0 }
//~^ ERROR `#[export]` attribute is only usable with `v0` mangling scheme
