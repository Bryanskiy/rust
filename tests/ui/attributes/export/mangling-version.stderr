error: `#[export]` attribute is only usable with `v0` mangling scheme
  --> $DIR/mangling-version.rs:7:1
   |
LL | pub extern "C" fn foo() -> i32 { 0 }
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

